### ISSUE TITLE
excluding false positives using regex and exclude.yml file

### ISSUE THREAD

**Issue by sateeshmaheshwaram** [feeltheajf/trufflehog3#60]
from trufflehog3 import DEFAULT_RULES_FILE
from trufflehog3.core import load_rules
from trufflehog3.models import File
from trufflehog3.render import text
from trufflehog3.search import search

content = """# Test content
password = "570a60db1d410c81e7ad8dfc7fdb429c"
print(password)
"""

rules = load_rules(DEFAULT_RULES_FILE)
issues = search(File(path="", content=content), rules=rules)
print(text(issues))

#######
its working as expected, but we would like to add exclude.yml file (which has few regexs) to eliminate false positives.
is there any way we could add along with above lines?
currently exclude.yml file is supplied while executing trufflehog3 -o exclude.yml.

**Comment by feeltheajf** [comment]
This should work:

```py
from trufflehog3.core import load_config

config = load_config(PATH_TO_CONFIG_FILE)
issues = search(File(path="", content=content), rules=rules, exclude=config.exclude)
```

In the future, please read the documentation before opening an issue: [feeltheajf.github.io/trufflehog3](https://feeltheajf.github.io/trufflehog3).
